website/netlify.toml

```diff
@@ -2,6 +2,14 @@
   base = "website/"
   publish = "build/"
   command = "npm run build"
+
+#https://answers.netlify.com/t/forwarding-subdomain-to-a-path/2988
+  
+[[redirects]]
+  from = "https://docs.openebs.io/*"
+  to = "https://openebs.io/docs/:splat"
```

Comment:
Thank you @nate-double-u for working on this one! 

Can we set this to redirect `"https://docs.openebs.io/*"` to a fixed location `"https://openebs.io/docs/`?

While `:splat` is great, it will not work with the current documentation changes. Along with the change in the domain for the openebs docs, we have also updated the docusaurus version from v1 to v2, which results in breaking changes for the file location. 

for example:
`https://docs.openebs.io/docs/next/features.html` changes to `https://openebs.io/docs/introduction/features`

Reply:
That makes sense, I can make that change.

Reply:
(the latest force push addresses this)